calc.rb

```diff
@@ -0,0 +1,66 @@
+answer = nil
+puts "Hi there"
+while answer == nil 
+  puts "Please enter one of the following operations, addition, subtraction, \nmultiplication, division, exponent, or modulo:"
+  operation = gets.chomp.downcase
+#gets user input
+  puts "Please enter the first number:"
+  user_num1 = gets.chomp
+  num1 = user_num1.to_f
+  
+  puts "Please enter the second number:"
+  user_num2 = gets.chomp
+  num2 = user_num2.to_f
+#test to see if user input is acutally a number
+  if user_num1.to_i == 0 && user_num1 != "0"
```

Comment:
I now see that's totally redundant. I could use just one variable keep it as a string until I'm done testing user input then once that passes change it to an integer.